tests/test_models.py

```diff
@@ -679,46 +683,22 @@ def _remove_non_models(elements):
     return rv
 
 
-class MessageWeightingTests(unittest.TestCase):
-    """unittests for message weighting."""
-
-    #: The number of entities
-    num_entities: int = 16
-
-    #: The number of triples
-    num_triples: int = 101
+class InverseInDegreeEdgeWeightingTests(EdgeWeightingTests):
+    """Tests for inverse in-degree weighting."""
 
-    def setUp(self) -> None:
-        """Initialize data for unittest."""
-        self.source, self.target = torch.randint(self.num_entities, size=(2, self.num_triples))
+    cls = pykeen.models.unimodal.rgcn.weightings.InverseInDegreeEdgeWeighting
```

Comment:
maybe `pykeen.models.unimodal.rgcn.weightings` should be moved into `pykeen.nn.weightings`

Reply:
done in 0e1f317109ae558d9345cbd7601925352a491bdc